### PR TITLE
[ADD] account_tax_report_no_zeroes: Account tax report without zeroes

### PR DESCRIPTION
**Pull request by pedrobaeza**
# Account tax report without zeroes

Remove lines with a zero tax amount in the generic taxes report.
# Usage

Print the tax report as usual in Accounting > Reporting > Taxes > Taxes Report,
but with this module, only lines with amounts will be printed.
# Known issues / Roadmap
- Let the user to decide if they want the zeroes in the report or not. For now,
  zeroes are always hidden.

### CODE REVIEW

**Comment by gurneyalex** [comment]
:+1:

**Comment by sbidoul** [comment]
@chrisdec this looks very interesting. Can you test it?

**Comment by pedrobaeza** [comment]
Can somebody review this?

File: account_tax_report_no_zeroes/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Account tax report without zeroes',
+    'version': '1.0',
+    'category': 'Custom modifications',
```
Comment (sbidoul) [inline account_tax_report_no_zeroes/__openerp__.py]: Is there no better category?

File: account_tax_report_no_zeroes/README.rst [hunk 2]
```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```
Comment (sbidoul) [inline account_tax_report_no_zeroes/README.rst]: Add target line

File: account_tax_report_no_zeroes/README.rst [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+Remove lines with a zero tax amount in the generic taxes report.
+
+Usage
+=====
+
+Print the tax report as usual in Accounting > Reporting > Taxes > Taxes Report,
+but with this module, only lines with amounts will be printed.
+
+Known issues / Roadmap
```
Comment (sbidoul) [inline account_tax_report_no_zeroes/README.rst:23]: Add runbot button

**Comment by sbidoul** [comment]
A few minor remarks, otherwise :+1:

**Comment by pedrobaeza** [comment]
Thanks for the review, @sbidoul. Your comments has been addressed, so please merge.

**Comment by sbidoul** [comment]
Thanks for this module!

**Pull request merged**